packages/core/src/State.ts

```diff
@@ -126,7 +126,7 @@ export class State<
   /**
    * An object mapping actor IDs to spawned actors/invoked services.
    */
-  public children: Record<string, ActorRef<any>>;
+  public children: Record<string, SpawnedActorRef<any>>;
```

Comment:
since [this](https://github.com/davidkpiano/xstate/pull/2275) got merged in we should revert back to using `ActorRef` (there are multiple places in this PR doing the opposite switch)